### ISSUE TITLE
Channel -> Help -> Yum

### ISSUE THREAD

**Issue by ruckc** [eclipse/packagedrone#80]
Got this error when clicking Channel -> Help -> Yum

```
Sep 01 21:46:35 packagedrone.example.com server[88065]: 21:46:35.450 [qtp783616999-30] WARN  o.e.p.r.w.internal.ErrorHandlerImpl - Handling UI error
Sep 01 21:46:35 packagedrone.example.com server[88065]: javax.servlet.ServletException: javax.el.PropertyNotFoundException: The class 'org.eclipse.packagedrone.repo.channel.ChannelInformation' does not have the property 'name'.
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jasper.runtime.PageContextImpl.doHandlePageException(PageContextImpl.java:951) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jasper.runtime.PageContextImpl.handlePageException(PageContextImpl.java:886) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp._jspService(org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp:91) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at javax.servlet.http.HttpServlet.service(HttpServlet.java:790) [javax.servlet_3.1.0.v201410161800.jar:3.1.0]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:411) ~[org.apache.jasper.glassfish_2.3.2.20160622-1310.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:473) ~[org.apache.jasper.glassfish_2.3.2.20160622-1310.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:377) ~[org.apache.jasper.glassfish_2.3.2.20160622-1310.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at javax.servlet.http.HttpServlet.service(HttpServlet.java:790) [javax.servlet_3.1.0.v201410161800.jar:3.1.0]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.equinox.jsp.jasper.JspServlet.service(JspServlet.java:155) ~[org.eclipse.equinox.jsp.jasper_1.0.500.20151105-1625.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at javax.servlet.http.HttpServlet.service(HttpServlet.java:790) [javax.servlet_3.1.0.v201410161800.jar:3.1.0]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:845) [org.eclipse.jetty.servlet_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:583) [org.eclipse.jetty.servlet_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.ops4j.pax.web.service.jetty.internal.HttpServiceServletHandler.doHandle(HttpServiceServletHandler.java:70) [org.ops4j.pax.web.pax-web-jetty_4.2.5.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:566) [org.eclipse.jetty.security_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:226) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1180) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.ops4j.pax.web.service.jetty.internal.HttpServiceContext.doHandle(HttpServiceContext.java:271) [org.ops4j.pax.web.pax-web-jetty_4.2.5.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:511) [org.eclipse.jetty.servlet_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1112) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.Dispatcher.forward(Dispatcher.java:199) ~[org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.Dispatcher.forward(Dispatcher.java:74) ~[org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.controller.ModelAndViewRequestHandler.render(ModelAndViewRequestHandler.java:174) ~[org.eclipse.packagedrone.web_1.0.0.20160623-1148.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.controller.ModelAndViewRequestHandler.process(ModelAndViewRequestHandler.java:131) ~[org.eclipse.packagedrone.web_1.0.0.20160623-1148.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.DispatcherServlet.service(DispatcherServlet.java:111) ~[org.eclipse.packagedrone.web_1.0.0.20160623-1148.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at javax.servlet.http.HttpServlet.service(HttpServlet.java:790) [javax.servlet_3.1.0.v201410161800.jar:3.1.0]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:837) [org.eclipse.jetty.servlet_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1689) [org.eclipse.jetty.servlet_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.dispatcher.internal.FilterTracker.doRunFilter(FilterTracker.java:134) [org.eclipse.packagedrone.web.dispatcher_1.0.0.20160203-1224.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.dispatcher.internal.FilterTracker.access$1(FilterTracker.java:117) [org.eclipse.packagedrone.web.dispatcher_1.0.0.20160203-1224.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.dispatcher.internal.FilterTracker$2.doFilter(FilterTracker.java:127) [org.eclipse.packagedrone.web.dispatcher_1.0.0.20160203-1224.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.sec.web.filter.SecurityFilter.processFilter(SecurityFilter.java:102) [org.eclipse.packagedrone.sec.web_1.0.0.20160203-1701.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.sec.web.filter.SecurityFilter.doFilter(SecurityFilter.java:75) [org.eclipse.packagedrone.sec.web_1.0.0.20160203-1701.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.dispatcher.internal.FilterTracker$TrackedFilter.doFilter(FilterTracker.java:51) [org.eclipse.packagedrone.web.dispatcher_1.0.0.20160203-1224.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.dispatcher.internal.FilterTracker.doRunFilter(FilterTracker.java:122) [org.eclipse.packagedrone.web.dispatcher_1.0.0.20160203-1224.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.dispatcher.internal.FilterTracker.access$1(FilterTracker.java:117) [org.eclipse.packagedrone.web.dispatcher_1.0.0.20160203-1224.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.dispatcher.internal.FilterTracker$2.doFilter(FilterTracker.java:127) [org.eclipse.packagedrone.web.dispatcher_1.0.0.20160203-1224.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.repo.web.googlebot.GoogleBotFilter.doFilter(GoogleBotFilter.java:50) [org.eclipse.packagedrone.repo.web.googlebot_1.0.0.20151104-1838.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.dispatcher.internal.FilterTracker$TrackedFilter.doFilter(FilterTracker.java:51) [org.eclipse.packagedrone.web.dispatcher_1.0.0.20160203-1224.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.dispatcher.internal.FilterTracker.doRunFilter(FilterTracker.java:122) [org.eclipse.packagedrone.web.dispatcher_1.0.0.20160203-1224.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.packagedrone.web.dispatcher.internal.FilterTracker.doFilter(FilterTracker.java:114) [org.eclipse.packagedrone.web.dispatcher_1.0.0.20160203-1224.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1676) [org.eclipse.jetty.servlet_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:581) [org.eclipse.jetty.servlet_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.ops4j.pax.web.service.jetty.internal.HttpServiceServletHandler.doHandle(HttpServiceServletHandler.java:70) [org.ops4j.pax.web.pax-web-jetty_4.2.5.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:548) [org.eclipse.jetty.security_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:226) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1180) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.ops4j.pax.web.service.jetty.internal.HttpServiceContext.doHandle(HttpServiceContext.java:271) [org.ops4j.pax.web.pax-web-jetty_4.2.5.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:511) [org.eclipse.jetty.servlet_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1112) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:80) [org.ops4j.pax.web.pax-web-jetty_4.2.5.jar:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:134) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.Server.handle(Server.java:524) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:319) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:253) [org.eclipse.jetty.server_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:273) [org.eclipse.jetty.io_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:95) [org.eclipse.jetty.io_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.io.SelectChannelEndPoint$2.run(SelectChannelEndPoint.java:93) [org.eclipse.jetty.io_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.executeProduceConsume(ExecuteProduceConsume.java:303) [org.eclipse.jetty.util_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.produceConsume(ExecuteProduceConsume.java:148) [org.eclipse.jetty.util_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume.run(ExecuteProduceConsume.java:136) [org.eclipse.jetty.util_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:671) [org.eclipse.jetty.util_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:589) [org.eclipse.jetty.util_9.3.10.v20160621.jar:9.3.10.v20160621]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at java.lang.Thread.run(Thread.java:745) [na:1.8.0_102]

Sep 01 21:46:35 packagedrone.example.com server[88065]: Caused by: javax.el.PropertyNotFoundException: The class 'org.eclipse.packagedrone.repo.channel.ChannelInformation' does not have the property 'name'.
Sep 01 21:46:35 packagedrone.example.com server[88065]: at javax.el.BeanELResolver.getBeanProperty(BeanELResolver.java:686) ~[com.sun.el.javax.el_3.0.0.jar:3.0.1-b08]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at javax.el.BeanELResolver.getValue(BeanELResolver.java:350) ~[com.sun.el.javax.el_3.0.0.jar:3.0.1-b08]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at javax.el.CompositeELResolver.getValue(CompositeELResolver.java:188) ~[com.sun.el.javax.el_3.0.0.jar:3.0.1-b08]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at com.sun.el.parser.AstValue.getValue(AstValue.java:139) ~[com.sun.el.javax.el_3.0.0.jar:3.0.1-b08]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at com.sun.el.parser.AstValue.getValue(AstValue.java:203) ~[com.sun.el.javax.el_3.0.0.jar:3.0.1-b08]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at com.sun.el.parser.AstEmpty.getValue(AstEmpty.java:66) ~[com.sun.el.javax.el_3.0.0.jar:3.0.1-b08]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at com.sun.el.parser.AstNot.getValue(AstNot.java:63) ~[com.sun.el.javax.el_3.0.0.jar:3.0.1-b08]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at com.sun.el.ValueExpressionImpl.getValue(ValueExpressionImpl.java:226) ~[com.sun.el.javax.el_3.0.0.jar:3.0.1-b08]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jasper.runtime.PageContextImpl.evaluateExpression(PageContextImpl.java:1016) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp._jspx_meth_c_if_0(org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp:147) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp.access$3(org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp:139) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp$help_jspHelper.invoke0(org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp:235) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp$help_jspHelper.invoke(org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp:299) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jsp.tag.web.main.main_tag._jspx_meth_c_otherwise_1(org.apache.jsp.tag.web.main.main_tag:965) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jsp.tag.web.main.main_tag._jspx_meth_c_choose_3(org.apache.jsp.tag.web.main.main_tag:908) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jsp.tag.web.main.main_tag.doTag(org.apache.jsp.tag.web.main.main_tag:258) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp._jspx_meth_h_main_0(org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp:109) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: at org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp._jspService(org.apache.jsp.bundle._90.WEB_002dINF.views.yum.help_jsp:84) ~[na:na]
Sep 01 21:46:35 packagedrone.example.com server[88065]: ... 67 common frames omitted
```

**Comment by ctron** [comment]
Which version of Package Drone did you use for that?

**Comment by ruckc** [comment]
0.14

**Comment by ctron** [comment]
Yes, but this one is not released yet. So which milestone/nightly version did you use?

**Comment by ctron** [comment]
Will be closed in 0.14.0!

Thanks for reporting!
